app/javascript/packs/tagFollows.jsx

```diff
@@ -46,11 +54,11 @@ function listenForButtonClicks() {
 function handleButtonClick({ target }) {
   let trigger;
 
-  if (target.classList.contains('follow-action-button')) {
+  if (target.classList.contains('js-follow-tag-button')) {
```

Comment:
Renamed these to be tag specific so that when we use FollowButtons on the same page they do not conflict and we're more clear on use cases.